### PR TITLE
[Test] build test handler

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Create an engine that everyone can simply iterate with and playing around without using model server

### CODE REVIEW

File: engines/python/setup/djl_python/test_model.py [hunk 1]
```diff
@@ -103,27 +131,85 @@ def create_npz_request(list, key: str = None) -> Input:
...
+    return kwargs
+
+
+class TestHandler:
+
+    def __init__(self,
+                 entry_point: Union[str, ModuleType],
+                 model_dir: str = None):
+        self.serving_properties = load_properties(model_dir)
+        self.serving_properties = update_properties_with_env_vars(
```
Comment (frankfliu) [inline engines/python/setup/djl_python/test_model.py:190]: env var is default value, we should call update_properties_with_env_vars first, then load properties

File: engines/python/setup/djl_python/test_model.py [hunk 2]
```diff
@@ -103,27 +131,85 @@ def create_npz_request(list, key: str = None) -> Input:
...
+            self.serving_properties)
+        if isinstance(entry_point, str):
+            os.chdir(model_dir)
```
Comment (frankfliu) [inline engines/python/setup/djl_python/test_model.py:193]: Should we do this outside of TestHandler?

File: serving/docker/partition/trt_llm_partition.py [hunk 3]
```diff
@@ -38,9 +37,9 @@ def update_kwargs_with_env_vars(kwargs):
     env_vars = os.environ
     for key, value in env_vars.items():
         if key.startswith("OPTION_"):
-            key = key[7:].lower()
-            if key == "entrypoint":
-                key = "entryPoint"
+            key = key.lower()
+            if key == "option.entrypoint":
```
[the remark this answers is not in the document]
Comment (lanking520) [reply serving/docker/partition/trt_llm_partition.py:41]: good catch, let me update this

**Comment by rohithkrn** [comment]
can you add a readme file with instructions to use?

**Review by frankfliu — APPROVED** [review]

**Pull request merged**